Source/TextExtractor.Agents.NUnit/ManagerJobTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿//using kCura.Talos.Utility;
+using NUnit.Framework;
+using TextExtractor.Helpers.Models;
+using TextExtractor.Helpers.NUnit;
+using TextExtractor.Helpers.NUnit.Dependencies;
+using TextExtractor.Helpers.NUnit.Dependencies.Seams;
+using TextExtractor.Helpers.NUnit.Fixtures;
+using TextExtractor.TestHelpers;
+using TextExtractor.TestHelpers.Fakes;
+
+namespace TextExtractor.Agents.NUnit
+{
+    //[ReportingSuite("KCD")]
+	[TestFixture]
+	public class ManagerJobTests : FakesFixture
+	{
+		//Note: this test is below one second because all fake data (and there's alot) is
+		//generated recursively in the AgentDependencies instantiation
+		[Description("Executes entire agent with fake connection to Relativity, should complete without exceptions")]
+        //[ReportingTest("32619ddc-a589-4e26-9f3f-529e037a8d4b")]
+		[Category(TestCategory.UNIT)]
+		[Test]
+		public void Execute()
+		{
+			var managerJob = GetSystemUnderTest();
+
+			Assert.DoesNotThrow(() => managerJob.Execute());
+		}
+
+		[Category(TestCategory.UNIT)]
+		[Test]
```

Comment:
Can we please add description for every test? something like this 
[Test(Description = "Verifies that the application exits if a user cancels the job between the time it was entered and the agent picking it up ")]